spec/lib/efile/nj/nj1040_spec.rb

```diff
@@ -21,33 +21,48 @@ def over_65_birth_year
       expect(instance.lines[:NJ1040_LINE_6].value).to eq(1000)
     end
 
-    context 'when filer is older than 65' do
-      let(:intake) do
-        create(:state_file_nj_intake,
-               primary_birth_date: Date.new(over_65_birth_year, 1, 1))
-      end
-
-      it 'checks the self 65+ checkbox and sets line 7 to 1000' do
-        expect(instance.lines[:NJ1040_LINE_7_SELF].value).to eq(true)
-        expect(instance.lines[:NJ1040_LINE_7_SPOUSE].value).to eq(false)
-        expect(instance.lines[:NJ1040_LINE_7].value).to eq(1000)
+    context "when filer is not blind" do
```

Comment:
[dust] This file might be more readable if we grouped related tests. Maybe with a parent context `blindness exemption` or something like that

Reply:
agree with this, and same for the older/younger than 65 ones as well. would be nice to group in a a `describe`. IMO describe is for groups of related tests and context is for specific situations, so "blindness exemption" should be a describe